exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -1273,5 +1295,11 @@ public boolean generatedKeyAlwaysReturned() throws SQLException {
     return super.generatedKeyAlwaysReturned();
   }
 
+  private ResultSet executeSql(String sql) throws SQLException {
```

Comment:
I looked into the implementation of this in mysql. Fixed it in similar manner. But it is still a query.
Is it a good way to send session manager object via RPC?